extractor/src/main/java/org/schabi/newpipe/extractor/utils/Parser.java

```diff
@@ -84,6 +84,11 @@ public static boolean isMatch(String pattern, String input) {
         return mat.find();
     }
 
+    public static boolean isMatch(Pattern pattern, String input) {
+        Matcher mat = pattern.matcher(input);
```

Comment:
```suggestion
        final Matcher mat = pattern.matcher(input);
```

Reply:
Im waiting for the day, where we can convert to kotlin. the keyword `final` should die. Takes up so much space for so little extra information